pointpats/tests/test_centrography.py

```diff
@@ -26,6 +28,10 @@ def setUp(self):
             ]
         )
 
+    @pytest.mark.skipif(
```

Comment:
Should there also be flag in `minimum_rotated_rectangle()` to raise or warn if GEOS 3.12.0 is not installed?

Reply:
Nah, it works. It just may produce slightly different outcome. The correct MRR is ensured by GEOS 3.12 or Shapely 2.0.2 with any GEOS.